tests/functional/conservation/conserv_3d.py

```
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)